policies/platform-policy.md

```diff
@@ -0,0 +1,69 @@
+# Platform Policy
+
+Platforms are classified as "primary", "secondary", "community"
+and "unadopted". Support for a new platform should only be added if it
+is being adopted as a primary, secondary or community platform.
+
+## Primary
+
+*Definition:* A platform that is regularly tested through project CI
+on a project owned and managed system.
+    
+New Pull Requests (PRs) should not be merged unless the primary
+platforms are showing as "green" in CI. If the CI breaks for a branch
+(such as for a stable version or master) then it should be fixed as a
+priority.
+
+[Current primary platforms](../policy-supplemental/primary-platforms.html)
```

Comment:
Doesn't this link need to be updated now (same with other links below)?

Reply:
Right.  Fixed